scanner/scanner.go

```diff
@@ -269,13 +269,48 @@ func humanTime(seconds int) string {
 	return s
 }
 
-// Scan performs a scan against the Log.
-// For each x509 certificate found, foundCert will be called with the
-// index of the entry and certificate itself as arguments.  For each precert
-// found, foundPrecert will be called with the index of the entry and the raw
-// precert string as the arguments.
+func (s *Scanner) logThroughput(treeSize int64, stop <-chan bool) {
+	const wndSize = 15
+	wnd := make([]int64, wndSize)
+	wndTotal := int64(0)
+
+	ticker := time.NewTicker(time.Second)
+	for slot, filled, prevCnt := 0, 0, int64(0); ; slot = (slot + 1) % wndSize {
+		select {
+		case <-stop:
+			break
```

Comment:
As Travis points out, this only breaks out of the `select` not the `for`

Reply:
Yes, I just fixed that.